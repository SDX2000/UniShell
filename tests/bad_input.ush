#!/usr/bin/env unishell
echo "================"
echo $SCRIPT_NAME
echo "================"

asdasd
stat
echo
set
#echo ======