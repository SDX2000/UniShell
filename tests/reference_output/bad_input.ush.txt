ERROR: Unknown command: bad_command
ERROR: (ArgumentError) (ArgumentError(...), 'No argument supplied.')
ERROR: (ArgumentError) (ArgumentError(...), 'Incorrect number of arguments specified')
================
bad_input.ush
================




